### ISSUE TITLE
Other Browser Extensions (Firefox, Safari, etc.)

### ISSUE THREAD

**Issue by ghost-bub** [politecho/politecho#8]
It would be beneficial if there was a version of this supported on other browser platforms.

**Comment by Vash63** [comment]
Agreed, supporting more than one platform would make this much more usable.
